Makefile

```diff
@@ -177,19 +177,20 @@ ifeq ($(SKIP_KVSTORES),"false")
 	$(QUIET)rm -rf /tmp/cilium-consul-certs
 endif
 
-generate-cov: ## Generate coverage report for Cilium unit-tests.
+generate-cov: ## Generate coverage report for Cilium integration-tests.
 	# Remove generated code from coverage
 	$(QUIET) grep -Ev '(^github.com/cilium/cilium/api/v1)|(generated.deepcopy.go)|(^github.com/cilium/cilium/pkg/k8s/client/)' \
 		coverage-all-tmp.out > coverage-all.out
 	$(QUIET)$(GO) tool cover -html=coverage-all.out -o=coverage-all.html
 	$(QUIET) rm coverage.out coverage-all-tmp.out
 	@rmdir ./daemon/1 ./daemon/1_backup 2> /dev/null || true
 
-init-coverage: ## Initialize converage report for Cilium unit-tests.
+init-coverage: ## Initialize converage report for Cilium integration-tests.
 	$(QUIET) echo "mode: count" > coverage-all-tmp.out
 	$(QUIET) echo "mode: count" > coverage.out
 
-unit-tests: start-kvstores ## Runs all unit-tests for Cilium.
+integration-tests: GO_TAGS_FLAGS+=integration_tests
+integration-tests: start-kvstores ## Runs all integration-tests for Cilium.
```

Comment:
Now that the target is renamed, is this `integration-tests` target also running the unit tests? How do we know that both types of tests are continuing to be run as part of the CI? Bear with me, the answer to this might be simple but I couldn't tell from briefly perusing the changes.

What I had in mind was two targets, one for integration tests and one for unit tests, and these targets run the specific type of test you're asking for, and then we make doubly sure that we aren't omitting any of these tests from the testing infrastructure (ie Travis for unprivileged, Jenkins for privileged tests)

Reply:
> Now that the target is renamed, is this `integration-tests` target also running the unit tests?

Yes. `go test ./...` runs all the unit tests. `go test -tags=integration_tests ./...` runs all the unit tests and the integration tests.

If we wanted separate unit and integration test targets we'd have to tag all the unit test files with a build tag so we can exclude them. I don't know if this is worth doing.